roles/config-nagios-server/defaults/main.yml

```diff
@@ -1 +1,46 @@
 ---
+install_nagios: "no"
+
+nagios_download_dir: /tmp
+nagios_version: 4.4.5
+nagios_download_name: "nagios-{{nagios_version}}.tar.gz"
+nagios_download_url: https://github.com/NagiosEnterprises/nagioscore/archive
+nagios_extracted_dir: "nagioscore-nagios-{{nagios_version}}"
+nagios_user: nagiosadmin
+nagios_password: password
+
+nagios_epel_repo_url: https://dl.fedoraproject.org/pub/epel
+nagios_epel_repo_rpm: epel-release-latest-7.noarch.rpm
+
+nagios_plugins_url: "https://github.com/nagios-plugins/nagios-plugins/archive"
+nagios_plugins_archive: "release-2.2.1.tar.gz"
```

Comment:
Should the archive name be hardcoded?

I would use a variable just like in the case of `nagios-{{nagios_version}}.tar.gz`.